db/seeds.rb

```diff
@@ -61,3 +61,25 @@
   communal_area: communal_area,
   priority: [priority1, priority2, priority3, priority4].sample
 )
+
+user = FactoryBot.create(:user)
+
+# Create a `defect.update` activity to simulate a change of status - updating
+# the activity more realistically (i.e. creating them as "outstanding" and
+# _then_ updating their status) is tricky, as it doesn't assign an `owner` to
+# the activity, so we're essentially lying about their status here, but it's
+# slightly closer to what actually happens.
+Defect.all.each do |defect|
+  next if defect.status.downcase == 'outstanding'
+
+  completed_statuses = %w[completed closed raised_in_error rejected]
+  if completed_statuses.include?(defect.status.downcase)
+    defect.update(actual_completion_date: Faker::Date.between('2019-01-01', 8.days.from_now))
```

Comment:
Is random dates the right thing to do? Do we guarantee that some will be in the future with this?

Reply:
I was really doing this for testing it out locally so I could see some dates in the past, it is a long range, you're right... This is only used for local development (e.g. to see a report), not in tests, so I think it's not a huge issue, but I'll see if there's a more sensible range.

Reply:
I've made the range smaller, I don't know how important this is, as it's just for testing locally, but we can maybe revisit it in future if necessary.